### PR TITLE
Aden 2121 wka.undefined

### PR DESCRIPTION
**Pull request by jacekjursza**
please **DO NOT MERGE** yet

### CODE REVIEW

File: extensions/wikia/AdEngine/AdEngine2ContextService.class.php [hunk 1]
```diff
@@ -59,6 +58,7 @@ public function getContext( Title $title, $skinName ) {
 					'wikiIsTop1000' => $wg->AdDriverWikiIsTop1000,
 					'wikiLanguage' => $langCode,
 					'wikiVertical' => $hubService->getCategoryInfoForCity( $wg->CityId )->cat_name,
+					'mappedVerticalName' => $this->getMappedVerticalName( $wg->CityId ) //wikiCategory replacement for AdLogicPageParams.js::getPageLevelParams
```
Comment (gabrys) [inline extensions/wikia/AdEngine/AdEngine2ContextService.class.php]: alphabetical order please

File: extensions/wikia/AdEngine/js/spec/AdLogicPageParams.spec.js [hunk 2]
```diff
@@ -113,10 +113,11 @@ describe('AdLogicPageParams', function () {
 		var params = getParams({
 			wikiCategory: 'category',
 			wikiDbName: 'dbname',
-			wikiLanguage: 'xx'
+			wikiLanguage: 'xx',
+			mappedVerticalName: 'mappedVertical'
```
Comment (gabrys) [inline extensions/wikia/AdEngine/js/spec/AdLogicPageParams.spec.js]: alphabetical order please

File: extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php [hunk 3]
```diff
@@ -168,7 +190,8 @@ public function testGetContext(
 		$expectedTargeting = [],
 		$expectedProviders = [],
 		$expectedForcedProvider = null,
-		$expectedSlots = []
+		$expectedSlots = [],
+		$expectedMappings = ['other', 'life']
```
Comment (gabrys) [inline extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php]: $...vertical and $expectedMappedVertical

File: extensions/wikia/AdEngine/AdEngine2ContextService.class.php [hunk 4]
```diff
@@ -77,6 +77,27 @@ public function getContext( Title $title, $skinName ) {
 		} );
 	}
 
+	private function getMappedVerticalName( $cityId ) {
+		$wikiVertical = WikiFactoryHub::getInstance()->getWikiVertical( $cityId );
+		if ( !empty( $wikiVertical[ 'short' ] ) ) {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/AdEngine2ContextService.class.php]: Redundant spaces around array key ([array syntax](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax]))

File: extensions/wikia/AdEngine/AdEngine2ContextService.class.php [hunk 5]
```diff
@@ -77,6 +77,27 @@ public function getContext( Title $title, $skinName ) {
...
+				'books' => 'ent',
+				'comics' => 'ent',
+				'lifestyle' => 'life',
+				'music' => 'ent',
+				'movies' => 'ent'
+			];
+			$newVerticalName = strtolower( $wikiVertical[ 'short' ] );
+			if ( !empty( $mapping[ $newVerticalName ] ) ) {
+				return $mapping[ $newVerticalName ];
+			}
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/AdEngine2ContextService.class.php:96]: Redundant spaces around array key ([array syntax](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax]))

**Comment by fraszczakszymon** [comment]
LGTM :+1:

**Pull request merged**